docker/synapse.Dockerfile

```diff
@@ -3,9 +3,12 @@ ARG SYTEST_IMAGE_TAG=buster
 FROM matrixdotorg/sytest:${SYTEST_IMAGE_TAG}
 
 ARG PYTHON_VERSION=python3
+
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
What does this do, and/or why wasn't it needed before?

Reply:
Aligning the two dockerfiles, and also related to the addition of apt-utils (as above).